legion/engine/physics/colliders/convexcollider.hpp

```diff
@@ -44,17 +46,21 @@ namespace legion::physics
         */
         void CheckCollisionWith(ConvexCollider* convexCollider, physics_manifold& manifold) override;
 
-
         void PopulateContactPoints(std::shared_ptr<PhysicsCollider> physicsCollider, physics_manifold& manifold) override
         {
             physicsCollider->PopulateContactPointsWith(this, manifold);
         }
 
         void PopulateContactPointsWith(ConvexCollider* convexCollider, physics_manifold& manifold) override;
 
+        void UpdateTightBoundingVolume(const math::mat4& transform) override
+        {
+            UpdateTightAABB(transform);
+        }
+
         /**@brief Given the current transform of the entity, creates a tight AABB of the collider;
         */
-        void UpdateTightAABB(math::mat4 transform)
+        void UpdateTightAABB(const math::mat4& transform)
```

Comment:
empty function?

Reply:
will be used later

Reply:
Can you add a comment that the function is empty for the time being